src/app/messages/pl/messages.json

```diff
@@ -73,7 +73,7 @@
   "i18n:nav:profile:text": "Konto",
   "i18n:notification:appmessage:dismiss": "OK",
   "i18n:notification:appmessage:header": "Powiadomienia",
-  "i18n:notification:cookies:body": "Używamy ciasteczek, które zapewniają komfort korzystania z platformy. Pozostanie na tej stronie jest równoznaczne z wyrażeniem zgody na użycie ciasteczek. Więcej informacji oraz opcję cofnięcia udzielonej zgody znajdziesz na <a href=\"/privacy-policy\" target=\"blank\">Privacy Policy</a>",
+  "i18n:notification:cookies:body": "Używamy ciasteczek, które zapewniają komfort korzystania z platformy. Pozostanie na tej stronie jest równoznaczne z wyrażeniem zgody na użycie ciasteczek. Więcej informacji oraz opcję cofnięcia udzielonej zgody znajdziesz w <a href=\"/privacy-policy\" target=\"blank\">Polityce Prywatności</a>",
```

Comment:
If you have time, can you also take a look at lines 34, 36, 37, 68, and 71? :grimacing: